apis/features/v1/features_types.go

```diff
@@ -63,6 +68,9 @@ type FeatureTrackerSpec struct {
 type FeatureTrackerStatus struct {
 	// +optional
 	Conditions *[]conditionsv1.Condition `json:"conditions,omitempty"`
+	// Phase describes the Phase of DSCInitializationStatus
+	// This is used by OLM UI to provide status information to the user
+	Phase string `json:"phase,omitempty"`
```

Comment:
How is that different from [`Condition.Reason`](https://github.com/openshift/custom-resource-status/blob/master/conditions/v1/types.go#L17) for which we have [`featurev1.FeaturePhase`](https://github.com/opendatahub-io/opendatahub-operator/blob/db3bf7b89e8eac79ea58d8ca5442e34cc7b2e1dc/apis/features/v1/features_types.go#L30-L36)?

If we need `Status.Phase` because that's what OLM requires let's unify those phases? Otherwise, it's a bit confusing.

Reply:
wronly copy-pasted comments form what we set in DSCI   [  // Phase describes the Phase of DSCInitializationStatus](https://github.com/opendatahub-io/opendatahub-operator/blob/db3bf7b89e8eac79ea58d8ca5442e34cc7b2e1dc/apis/dscinitialization/v1/dscinitialization_types.go#L78)

main reason to add this is to get a value of "Progressing", "Error" or "Ready"
instead of showing detail of internal steps during Apply

if query FT CR, Status.Phase will be the only indication.

SetStatusCondition() does append to []Conditions, i think this is why from "before" screenshot it has both `Progressing,Available` there